src/type_methods.jl

```diff
@@ -101,6 +101,12 @@ function add_methods(rot_type)
                     call(::Type{$(rot_type)}, $(int_cons1...)) = call($(rot_type), $(int_cons2...))
                 )
     push!(qb.args, int_expr)
+    if (length(rot_type.parameters) == 2)  # it has an order
```

Comment:
This seems really obscure.  You could just check `if rot_type in [EulerAngles, ProperEulerAngles]` or something?